Services/JWSProvider/DefaultJWSProvider.php

```diff
@@ -45,16 +45,22 @@ class DefaultJWSProvider implements JWSProviderInterface
      * @var int
      */
     private $clockSkew;
+    
+    /**
+     * @var bool
+     */
+    private $validateTokensWithoutTtl;
```

Comment:
This class should not get new features, it is deprecated. Only LcobucciJWSProvider should benefit from this change

Reply:
> This class should not get new features, it is deprecated. Only LcobucciJWSProvider should benefit from this change

Understood. I have reverted all changes to DefaultJWSProvider. The additional parameter in the constructor of LcobucciJWSProvider should not cause an issue as the constructor is not defined in the providerInterface.